README.md

```diff
@@ -378,3 +378,7 @@ The following issues are currently being explored and input from the community i
 - Tony Kovanen ([@tonykovanen](https://twitter.com/tonykovanen)) – ▲ZEIT
 - Guillermo Rauch ([@rauchg](https://twitter.com/rauchg)) – ▲ZEIT
 - Dan Zajdband ([@impronunciable](https://twitter.com/impronunciable)) – Knight-Mozilla / Coral Project
+
+## License
+
+MIT
```

Comment:
@iamstarkov yeah, I agree the LICENSE file should be in the repo, but I think we don't need it added to the README.

BTW, the license is also declared in [package.json](https://github.com/zeit/next.js/blob/master/package.json#L6)